### PR TITLE
[nit] Fix a couple incorrect uses of msg_error.

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

File: library/summary.ml [hunk 1]
```diff
@@ -108,7 +108,7 @@ let unfreeze_summaries fs =
     with e when CErrors.noncritical e ->
       let e = CErrors.push e in
       Feedback.msg_error
-        Pp.(seq [str "Error unfrezing summay %s\n%s\n%!";
+        Pp.(seq [str "Error unfrezing summary %s\n%s\n%!";
```
Comment (maximedenes) [inline library/summary.ml]: You missed another typo on the same line ;)

Comment (ejgallego) [inline library/summary.ml]: well spotted, I will update the pr

**Comment by maximedenes** [comment]
LGTM

**Comment by herbelin** [comment]
Sorry, I come late, but `Show Thesis` could simply be removed (never implemented, not documented, associated to the declarative mode).

**Pull request merged**